VAMobile/package.json

```diff
@@ -10,6 +10,7 @@
     "test:watch": "jest --coverage --colors --watchAll",
     "test:debug": "yarn run env:test && jest --colors",
     "test": "yarn run env:test && jest --coverage --colors --silent",
+    "test:local": "yarn run env:test && jest",
```

Comment:
Is "local" the right tag for this? Could it be something like "no-coverage" or was there a specific reason this was needed?

Reply:
Yeah, it could be named something better. 

This came out the that the coverage report was a a Long Time. and was running for all the tests even if we were only testing 1 file (or a subset). So this was created to help tighten that feedback loop